src/stim/simulators/tableau_simulator.pybind.cc

```diff
@@ -2019,4 +2020,31 @@ void stim_pybind::pybind_tableau_simulator_methods(pybind11::module &m, pybind11
                 )
         )DOC")
             .data());
+    c.def(
+        "reference_sample",
```

Comment:
I don't think this method makes sense on the `stim.TableauSimulator` class. The issue is that it is very stateful, for example the user may have applied Hadamard gates. What does it mean to ask for a reference sample from the last half of a circuit? Its contents can depend very sensitively on the beginning of the circuit.

Instead of adding it here, add it as a method on `stim.Circuit`.